packages/create-cloudflare/templates/openapi/c3.json

```diff
@@ -0,0 +1,10 @@
+{
+	"configVersion": 1,
+	"id": "openapi",
+	"displayName": "OpenAPI 3.1",
```

Comment:
Let's rename this to `API starter (OpenAPI compliant)` just to give some more context for users getting started.